octopoes/octopoes/api/router.py

```diff
@@ -524,7 +527,38 @@ def recalculate_bits(octopoes: OctopoesService = Depends(octopoes_service)) -> i
 
     octopoes.commit()
 
-    return inference_count
+    RECALCULATE_BITS_SESSIONS[session_id] = inference_count
+
+
+@router.post("/bits/recalculate", tags=["Bits"])
+async def recalculate_bits(octopoes: OctopoesService = Depends(octopoes_service)) -> Any:
+    session_id = uuid.uuid4()
+    RECALCULATE_BITS_SESSIONS[session_id] = -1
+
+    asyncio.create_task(recalculate_bits_launcher(octopoes, session_id))
+
+    return {"session_id": session_id}
+
+
+@router.post("/bits/recalculate/status", tags=["Bits"])
+async def recalculate_bits_status(request: Request) -> Any:
+    try:
+        data = await request.body()
+    except Exception as e:
+        raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail="Error receiving objects") from e
+    if data:
+        session_id = uuid.UUID(data.decode())
```

Comment:
Would be nice in a url parameter if we use a GET so that we can just add it to the arguments with a proper UUID type and let FastAPI handle the rest.

Reply:
See comment above.